src/kakarot/instructions/system_operations.cairo

```diff
@@ -1086,26 +1099,17 @@ namespace CreateHelper {
             tempvar is_prefix_not_0xef = is_not_zero(0xef - [evm.return_data]);
         }
 
-        let is_reverted = is_not_zero(evm.reverted);
-        let checks_success = enough_gas * code_size_limit * is_prefix_not_0xef;
-        let success = (1 - is_reverted) * checks_success;
+        let success = enough_gas * code_size_limit * is_prefix_not_0xef;
 
         // Stack output: the address of the deployed contract, 0 if the deployment failed.
         let (address_high, address_low) = split_felt(evm.message.address.evm * success);
         tempvar address = new Uint256(low=address_low, high=address_high);
-
         Stack.push(address);
 
         if (success == FALSE) {
-            // The only case where a create opcode has returndata is if it reverted with the REVERT opcode.
-            tempvar is_exceptional_revert = is_not_zero(Errors.REVERT - evm.reverted) + (
-                1 - checks_success
-            );
-            let return_data_len = (1 - is_exceptional_revert) * evm.return_data_len;
-
             tempvar evm = new model.EVM(
                 message=evm.message.parent.evm.message,
-                return_data_len=return_data_len,
+                return_data_len=0,
```

Comment:
what is happening in this case?
success is false, and revert is also false?

Reply:
the execution of the create was successful, but the checks before writing the code in the destination accounts were not.